pro/src/pages/IndividualOffer/commons/constants.ts

```diff
@@ -0,0 +1,3 @@
+export enum OFFER_LOCATION {
```

Comment:
Je serai pour mettre les enum dans des fichiers enums.ts mais on peut en reparler

Reply:
Ce type de fichier n'existe pas encore dans la codebase mais on peut en reparler oui !
Je laisse as it is pour le moment.